app/cli/cmd/attached_integration_list.go

```diff
@@ -53,22 +53,32 @@ func attachedIntegrationListTableOutput(attachments []*action.AttachedIntegratio
 	fmt.Println("Integrations attached to workflows")
 	t := newTableWriter()
 	t.AppendHeader(table.Row{"ID", "Kind", "Config", "Attached At", "Workflow"})
-	for _, i := range attachments {
-		wf := i.Workflow
-		integration := i.Integration
+	for _, attachment := range attachments {
+		wf := attachment.Workflow
+		integration := attachment.Integration
+
+		// Merge attachment and integration configs
```

Comment:
Are we merging integration and attachment configs or we are merging configurations provided during **registration** and attachment?

Reply:
Yeah, I am nitpicking here a bit, but I was not sure at first what you mean by the **integration config** here.